pyteal/ast/itxn.py

```diff
@@ -200,6 +206,111 @@ def SetFields(cls, fields: Dict[TxnField, Union[Expr, List[Expr]]]) -> Expr:
         fieldsToSet = [cls.SetField(field, value) for field, value in fields.items()]
         return Seq(fieldsToSet)
 
+    @classmethod
+    def MethodCall(
+        cls,
+        *,
+        app_id: Expr,
+        method_signature: str,
+        args: List[Union[abi.BaseType, Expr, Dict[TxnField, Union[Expr, List[Expr]]]]],
```

Comment:
```suggestion
        args: list[abi.BaseType | Expr | dict[TxnField, (Expr | list[Expr])]]]],
```
not sure about the last `(Expr | list[Expr])`

3.10 suggestion